cmd/subctl/reconstruct.go

```diff
@@ -0,0 +1,100 @@
+package subctl
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/admiral/pkg/reporter"
+	"github.com/submariner-io/subctl/internal/cli"
+	"github.com/submariner-io/subctl/internal/constants"
+	"github.com/submariner-io/subctl/internal/exit"
+	"github.com/submariner-io/subctl/internal/restconfig"
+	//"github.com/submariner-io/subctl/pkg/broker"
+	"github.com/submariner-io/subctl/pkg/cluster"
+)
+
+var recoverRestConfigProducer = restconfig.NewProducer().WithPrefixedContext("broker").WithDefaultNamespace(constants.OperatorNamespace).
+	WithPrefixedNamespace("broker", constants.DefaultBrokerNamespace)
+
+var (
+	ipsecFile  string
+)
+
+// recoverBrokerInfo represents the reconstruct command.
```

Comment:
For consistency, please use “recover” rather than “reconstruct” (and `recover.go`).